packages/ui-admin/src/workspace/bots/CreateBotModal.tsx

```diff
@@ -113,13 +121,22 @@ class CreateBotModal extends Component<Props, State> {
     }
     this.setState({ isProcessing: true })
 
-    const newBot = {
+    const newBot: Partial<BotConfig> & { template: Partial<BotTemplate> } = {
       id: this.state.botId,
       name: this.state.botName,
       template: _.pick(this.state.selectedTemplate, ['id', 'moduleId']),
       category: this.state.selectedCategory && this.state.selectedCategory.value
     }
 
+    if (this.state.isCloudBot) {
+      newBot.isCloudBot = this.state.isCloudBot
+      newBot.cloud = {
+        clientId: this.state.cloudClientId,
+        clientSecret: this.state.cloudClientSecret,
+        oauthUrl: 'https://oauth.botpress.dev/oauth2/token'
```

Comment:
I'm not sure to like having a default buried in the frontend code... I'd suggest moving that URL to `common/detaults.ts` for now. We could use it in the backend instead of the frontend, and maybe overridable with an environment variable?